pkg/pillar/cast/cast.go

```diff
@@ -463,3 +463,17 @@ func CastFlowStatus(in interface{}) types.IPFlow { //revive:disable-line
 	}
 	return output
 }
+
+// CastPhysicalIOAdapterList : Cast interface type into
+//       types.PhysicalIOAdapterList
+func CastPhysicalIOAdapterList(in interface{}) types.PhysicalIOAdapterList { //revive:disable-line
+	b, err := json.Marshal(in)
+	if err != nil {
+		log.Errorf("json Marshal in PhysicalIOAdapterList, %v", err)
```

Comment:
Should be log.Fatal*

Reply:
Fixed..

Reply:
I guess I copied from CastFlowStatus. I changed CastFlowStatus to also use Fatalf..

Reply:
FWIW The only place we need Error is when the user can edit a json file, such as the DevicePortConfig which comes from a USB stick.